_pages/dev/components/shared-components/table-component.md

```diff
@@ -0,0 +1,140 @@
+---
+title: Table Component
+feature:
+- name: Table Component
+  spa_version: 3.0
+  cx_version: n/a
+---
+
+{% capture version_note %}
+{{ site.version_note_part1 }} 3.0 {{ site.version_note_part2 }}
+{% endcapture %}
+
+{% include docs/feature_version.html content=version_note %}
+
+The table component is a low-level component for rendering tabular data, using the standard HTML `<table>` element. While HTML tables can have very complex structures, the table component implementation is fairly limited.
+
+The main purpose of the table component is to allow you to generate tables based on the header configuration, and to provide a mechanism to customize the rendering of table cells.
+
+The table component supports various layout orientations, as well as responsive configurations for table definitions.
+
+## Table Cell Configuration
+
+Table configurations are defined by a table type. The type is a unique key that is used to identify the table configuration from the `TableConfig`.
+
+The `TableConfig` contains a global configuration for all tables, as well as type-specific configurations. The global configuration defines the default components to render the header and data cells. These components can be overridden in type-specific configurations.
+
+The following is a basic example of a table configuration, which provides the cell definition for the "budget" table:
+
+```ts
+provideConfig({
+  table: {
+    'my-table': {
+      cells: ['name', 'active', 'amount', 'dateRange', 'unit'],
+    },
+  },
+} as TableConfig);
+```
+
+## Table Layout Configuration
+
+The table can be rendered in the following different layout orientations:
+
+- `VERTICAL` renders the table vertically, with a heading on top of the table
+- `VERTICAL_STACKED` renders a horizontally-oriented table layout, with the table headers in the first column of the table
+- `HORIZONTAL` renders the table horizontally, with the header on the left side of the table
+
+The vertical layout is the default layout.
+
+A layout configuration can be provided for each table type, as shown in the following example:
+
+```ts
+provideConfig({
+  table: {
+    'my-table': {
```

Comment:
I've pushed a new version with updated code examples to align the (sample) names